extensions/amp-accordion/amp-accordion.md

```diff
@@ -86,6 +89,10 @@ Set this attribute on the `<amp-accordion>` to opt out of preserving the collaps
 
 Set this attribute on a `<section>` to display the section as expanded on page load.
 
+#####  expand-single-section
```

Comment:
please add ticks ` around the attribute name

Reply:
Done and did this for the other 2 attributes as well.